php-classes/Slate/Connectors/AbstractSpreadsheetConnector.php

```diff
@@ -1496,6 +1495,15 @@ protected static function _applyUserChanges(IJob $Job, User $User, array $row, a
         }
     }
 
+    protected static function _applyStudentUserChanges(IJob $Job, User $User, array $row, array &$results)
+    {
+        static::_applyUserChanges($Job, $User, $row, $results);
+
+        if ($User->AccountLevel == 'Disabled') {
+            $User->AccountLevel = 'User';
+        }
```

Comment:
I don't think we want this logic to contradict any value in the sheet, this should just be in an `else` within `_applyUserChanges` if there is no provided accountlevel

Reply:
nvm